lib/modules/apostrophe-pieces/public/js/manager-modal.js

```diff
@@ -723,7 +712,6 @@ apos.define('apostrophe-pieces-manager-modal', {
       var $selectEverything = self.getSelectEverything();
       if ($pieces.length === $checked.length) {
         self.getSelectAll().prop('checked', true);
-        self.shrinkGrid();
```

Comment:
We can't just remove methods because third parties may be expecting them (bc break). If they are no longer needed they could become empty methods.